VERSION

```diff
@@ -0,0 +1 @@
+2.7.2+rev1
```

Comment:
Please change to meta-resin v2.7.4 and use this CHANGELOG style: https://github.com/resin-os/resin-raspberrypi/blob/master/CHANGELOG.md